packages/migrations/src/migration.ts

```diff
@@ -96,30 +129,8 @@ export async function runMigrationsAsync(provider: Provider, txDefaults: Partial
     await web3Wrapper.awaitTransactionSuccessAsync(
         await exchange.registerAssetProxy.sendTransactionAsync(erc721Proxy.address),
     );
-
-    // Dummy ERC20 tokens
-    for (const token of erc20TokenInfo) {
-        const totalSupply = new BigNumber(1000000000000000000000000000);
-        // tslint:disable-next-line:no-unused-variable
-        const dummyErc20Token = await wrappers.DummyERC20TokenContract.deployFrom0xArtifactAsync(
-            artifacts.DummyERC20Token,
-            provider,
-            txDefaults,
-            token.name,
-            token.symbol,
-            token.decimals,
-            totalSupply,
-        );
-    }
-
-    // ERC721
-    // tslint:disable-next-line:no-unused-variable
-    const cryptoKittieToken = await wrappers.DummyERC721TokenContract.deployFrom0xArtifactAsync(
-        artifacts.DummyERC721Token,
-        provider,
-        txDefaults,
-        erc721TokenInfo[0].name,
-        erc721TokenInfo[0].symbol,
+    await web3Wrapper.awaitTransactionSuccessAsync(
```

Comment:
We need to also register the erc20 and erc721 proxies within the MAP. 

Weprobably want to deploy the AssetProxyOwner after the MAP so that we can include the MAP address in the constructor args.